src/api/api.ts

```diff
@@ -1 +1,7 @@
-export const BASE_URL = 'https://mate.academy/students-api';
+export const BASE_URL = 'https://mate.academy/students-api/';
+
+export async function getAllUsers() {
```

Comment:
Provide an explicit return type in a function declaration.
```suggestion
export async function getAllUsers(): Promise<User[]> {
```